lib/client/rpc/proposals.ts

```diff
@@ -25,20 +30,24 @@ import {
 import { TransactionSender } from "@/transactions";
 import { enrichTokenMetadata } from "@/tokens";
 import { getProposalFromAccount } from "@/proposal";
+import { AMM_PROGRAM_ID, AUTOCRAT_PROGRAM_ID, AutocratClient, CONDITIONAL_VAULT_PROGRAM_ID, ConditionalVault, ConditionalVaultClient, getATA, getAmmAddr, getDaoTreasuryAddr, getVaultAddr, getVaultFinalizeMintAddr, getVaultRevertMintAddr } from "@metadaoproject/futarchy-ts";
+import { MethodsBuilder } from "@coral-xyz/anchor/dist/cjs/program/namespace/methods";
+import { TOKEN_PROGRAM_ID, ASSOCIATED_TOKEN_PROGRAM_ID } from "@solana/spl-token";
 
 export class FutarchyRPCProposalsClient implements FutarchyProposalsClient {
-  private rpcProvider: Provider;
+  private rpcProvider: AnchorProvider;
   private futarchyProtocols: FutarchyProtocol[];
   private transactionSender: TransactionSender | undefined;
-
+  private autocratClient: AutocratClient
   constructor(
-    rpcProvider: Provider,
+    rpcProvider: AnchorProvider,
     futarchyProtocols: FutarchyProtocol[],
     transactionSender: TransactionSender | undefined
   ) {
     this.rpcProvider = rpcProvider;
     this.futarchyProtocols = futarchyProtocols;
     this.transactionSender = transactionSender;
+    this.autocratClient = new AutocratClient(rpcProvider, AUTOCRAT_PROGRAM_ID, CONDITIONAL_VAULT_PROGRAM_ID, AMM_PROGRAM_ID, [])
```

Comment:
you will need to initialize this based on the futarchyProtocol of the dao you are trying to create a proposal on... I assume the proposal will be created on a specific autocrat version

Reply:
the autocrat client is only for v1 for now

Reply:
oh I see... so I guess all proposals will just be created on v1 going forward I suppose?